source/javascripts/lib/wirsing/filters/switch.coffee

```diff
@@ -0,0 +1,17 @@
+@wirsing ?= {}
+wirsing.filter ?= {}
```

Comment:
I think we should create a simple `namespace` function that creates that for us. In the long run this will be much easier for reason about.

Something like this: `namespace 'wirsing.filter'`